DeAceroDLabs/marketplace-frontend
# Feature/dmp 444

## What is done

[[DMP-444] Phone Field](https://deacero.atlassian.net/browse/DMP-444)

## How is done

- Created a `PhoneField` component that will be used to validate Phone fields, with custom error message.

## How to test

1. Run the project via `npm start`
2. /signup
3. Write a number in any phone field.
4. See the error messages.
